ansible/roles/ssh/tasks/main.yaml

```diff
@@ -22,5 +22,14 @@
     - { src: "{{jenkins_public_ssh_key_file_path}}",
         dest: "{{jenkins_home_dir}}/.ssh/authorized_keys",
         owner: jenkins, group: jenkins, mode: "0600" }
+
+- name: Add Github and upload server's host key
```

Comment:
I think we can remove this paragraph from the README: "add the target host to ~/.ssh/known_hosts in the source host. It is also possible to do it by executing a simple SSH command from the source host to the target host. Alternatively, you could disable "StrictHostKeyChecking" for all SSH connections in the source host, but that may be a security hazard."

The prompt should be descriptive enough so the user knows what to do.